src/cards/ha-camera-card.html

```diff
@@ -49,76 +51,78 @@
 </dom-module>
 
 <script>
-Polymer({
-  is: 'ha-camera-card',
-  UPDATE_INTERVAL: 10000, // ms
-
-  properties: {
-    hass: {
-      type: Object,
-    },
-
-    stateObj: {
-      type: Object,
-      observer: 'updateCameraFeedSrc',
-    },
-
-    cameraFeedSrc: {
-      type: String,
-    },
-
-    imageLoaded: {
-      type: Boolean,
-      value: true,
-    },
-
-    /**
-     * The z-depth of the card, from 0-5.
-     */
-    elevation: {
-      type: Number,
-      value: 1,
-      reflectToAttribute: true,
-    },
-  },
-
-  listeners: {
-    tap: 'cardTapped',
-  },
-
-  attached: function () {
-    this.timer = setInterval(
-      function () {
-        this.updateCameraFeedSrc(this.stateObj);
-      }.bind(this),
-      this.UPDATE_INTERVAL
-    );
-  },
-
-  detached: function () {
+const UPDATE_INTERVAL = 10000; // ms
+
+class HaCameraCard extends window.hassMixins.EventsMixin(Polymer.Element) {
+  static get is() { return 'ha-camera-card'; }
+  static get properties() {
+    return {
+      hass: {
+        type: Object,
+      },
+
+
+      stateObj: {
+        type: Object,
+        observer: 'updateCameraFeedSrc',
+      },
+
+      cameraFeedSrc: {
+        type: String,
+      },
+
+      imageLoaded: {
+        type: Boolean,
+        value: true,
+      },
+
+      /**
+       * The z-depth of the card, from 0-5.
+       */
+      elevation: {
+        type: Number,
+        value: 1,
+        reflectToAttribute: true,
+      },
+    };
+  }
+
+  ready() {
+    super.ready();
+    this.addEventListener('tap', () => this.cardTapped());
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    this.timer = setInterval(() => this.updateCameraFeedSrc(), UPDATE_INTERVAL);
+  }
+
+  disconnectedCallback() {
+    super.disconnectedCallback();
     clearInterval(this.timer);
-  },
+  }
 
-  cardTapped: function () {
+  cardTapped() {
     this.fire('hass-more-info', { entityId: this.stateObj.entity_id });
-  },
+  }
 
-  updateCameraFeedSrc: function (stateObj) {
-    var attr = stateObj.attributes;
+  updateCameraFeedSrc() {
```

Comment:
Observers get the latest value passed in. We should use that one instead of using `this`

Reply:
Oh nevermind, realized that we use this in connectedCallback too.